### DISCUSSION TITLE
Label Suggestion: "Run TestConnection function" command

### DISCUSSION THREAD

**Discussion by bgribaudo** [microsoft/vscode-powerquery-sdk#183] [Ideas]
For what it's worth...idea: Rename to "Validate TestConnection" (shorter, which is in alignment with most other command names).

**Comment by ptyx507x** [comment]
thank you for posting this!

The initial feedback that we had when the label was just "TestConnection" is that it wasn't specific enough. It led to believe some users that they were testing something more among the lines of a credential challenge rather than if their implementation of the TestConnection function was correct and that's the reason why we have the label that we have today.

Nevertheless, we're completely open to new ideas. I'll leave this discussion open and hoping that more folks can upvote the idea or suggest any other labels for this particular command.
